src/app/components/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,124 @@
+
+
+.sidenav {
+   
+    background:#a99892;
+    transition: all 5s ease;
+    z-index: 1;
+    top: 0;
+    width: 5rem;
+    height: 95vh;
+    box-shadow: 0.063rem 0 1.25rem 0 #8690a3;
+    position: fixed;
+    padding-top: 140px; 
+
+    .logo-container{
+        display: flex;
+        align-items :center;
+        padding: 0.938rem 0.938rem 0.938rem ;
+        width: 100%;
+    .LU{
+        width: 60px;
+        height: 60px;
+       
+        object-fit: cover;
+    }
+    
+    .logo{
+        background:#a99892;
```

Comment:
This color is present in _imports.scss as $navbar-color. Use _imports.scss colors.